### PR TITLE
Update element mappings for newly released elements

### PR DESCRIPTION
**Pull request by rictic**

### CODE REVIEW

File: lib/element_mapping.js [hunk 1]
```diff
@@ -166,7 +166,11 @@ module.exports = {
     noReplacement: true,
   },
   'core-list': {
-    noReplacement: true,
+    name: 'iron-list',
+    notice: 'iron-list no longer tracks selection of its elements. Also the ' +
+            'data binding of the template child has changed some to be more ' +
+            'like dom-repeat. See: ' +
```
Comment (justinfagnani) [inline lib/element_mapping.js]: mention adding `as="model"` to fix bindings. Add TODO to add that automatically.

File: lib/element_mapping.js [hunk 2]
```diff
@@ -292,7 +296,12 @@ module.exports = {
     }
   },
   'core-tooltip': {
-    noReplacement: true
+    name: 'paper-tooltip',
+    notice: 'Whereas core-tooltip was intended to contain the element it ' +
```
Comment (justinfagnani) [inline lib/element_mapping.js]: Make this notice more ominous? It'll probably add the tooltip to the wrong element.

Comment (rictic) [inline lib/element_mapping.js]: Done.

File: lib/element_mapping.js (hunk 1 above)
Comment (rictic) [inline lib/element_mapping.js]: :+1:, done

**Comment by justinfagnani** [comment]
lgtm

**Pull request merged**